wcfsetup/install/files/style/bootstrap/mixin/mediaQuery.scss

```diff
@@ -56,7 +56,25 @@
 
 /* desktop */
 @mixin screen-lg() {
+	@media screen and (min-width: 1025px) and (max-width: 1280px) {
```

Comment:
This change would completely break backwards compatibility, because `screen-lg` would no longer apply to screen widths beyond `1280px`.

- `screen-lg` implicitly works like `screen-lg-up`
- `screen-lg-only` works between `1025px` and `1280px`
- `screen-lg-down` works as proposed
- `screen-xl` works as proposed

This does break with the naming scheme, but this is the lesser evil.

Reply:
Applied the requested changes. I was unsure, if i should risk a breaking change in favor of the naming convention, or not.